### PR TITLE
Update README, index.rst

### PR DESCRIPTION
**Pull request by v-h-gn**
### Info
* Renames Contributing section to Info.
* Adds link to GitHub
* Adds General Welcome

### N/A
* Adds clarification information in regards to the directory as well as structure of wiki information.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -3,16 +3,12 @@ ProjectKorra Official Wiki
 ==========================
 
 .. image:: https://readthedocs.org/projects/projectkorrawiki/badge/?version=latest
-    :target: https://projectkorrawiki.readthedocs.io/en/latest/?badge=latest
+    :target: https://wiki.projectkorra.com/en/latest/
```
Comment (ChristopherWMM) [inline README.rst]: Should be `https://wiki.projectkorra.com/`.

**Review by ChristopherWMM — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (vtovmasian) [reply README.rst]: It is?

Comment (ChristopherWMM) [reply README.rst]: No, you really shouldn't be adding the `/en/latest/` to the end of the link URL.

Comment (vtovmasian) [reply README.rst]: Oh thats what you meant, gotcha

**Review by ChristopherWMM — APPROVED** [review]
Looks good 👍

**Comment by v-h-gn** [comment]
:dab:

**Review by Simplicitee — APPROVED** [review]

**Pull request merged**